install.sh

```diff
@@ -7,8 +7,14 @@ ORACLE_RPM="$(basename $ORACLE_FILE .zip)"
 
 cd "$(dirname "$(readlink -f "$0")")"
 
-sudo apt-get -qq update
-sudo apt-get --no-install-recommends -qq install bc libaio1 rpm unzip
+deps="bc libaio1 rpm unzip"
+if dpkg -s $deps >/dev/null 2>/dev/null; then
+  echo "Oracle XE dependencies are already installed: $deps"
+else
+  echo "Installing Oracle XE dependencies: $deps"
+  sudo apt-get -qq update
+  sudo apt-get --no-install-recommends -qq install $deps
```

Comment:
Here are two examples of people using travis-oracle using envvars from repository settings instead of `.travis.yml`
https://travis-ci.org/valdasraps/cmsdbldr/builds/112850069
https://travis-ci.org/jksy/oci8-auto-binder/jobs/100054975